### PR TITLE
Compass client

### PR DESCRIPTION
**Pull request by dem4ply**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -45,3 +45,6 @@ venv
 *.egg*
 
 .env
+
+
+tests/compass/cassettes/Test_send_orders_local.test_example_*
```
Comment (sojant) [inline .gitignore]: This seems like a legit path, do you really want to git ignore this ?

File: tetrapod/compass/client.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            raise exceptions.Bad_formatted_xml from e
+
+        order = clean_result[ 'mvrdl' ][ 'order' ]
+        if isinstance( order, list ):
+            logger_send_orders.warning(
+                "was receiving a query and not the expected response" )
+            order = order[1]
+        self._validate_response( order )
+        if 'error' in str( order ).lower():
+            import pdb
```
Comment (sojant) [inline tetrapod/compass/client.py]: We won't want this in PROD

File: tests/compass/client.py [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+
+    def test_example_four( self ):
+        self.client.send_orders(
+            first_name='PRETEND', last_name='AKKEREN', state='GA',
+            driver_license_number='999999999',
+            date_of_birth=datetime.date( 1988, 9, 12 ), gender='', )
+
+
+@unittest.skip( 'this should be only run in local' )
+class Test_send_orders_local( VCRTestCase, Test_compass ):
```
Comment (sojant) [inline tests/compass/client.py]: This test is skipped, it uses some chibi reference and a pdb.post_mortem.
Do you really mean to add this test?

Comment (dem4ply) [reply tests/compass/client.py]: yes, the real tests is in the service
in this proyect i dont put the cassetes is only for local tests becuase take more of 10 minutes only that test, that is why i left the post mortem

File: .gitignore (hunk 1 above)
Comment (dem4ply) [reply .gitignore]: yes becuase are more of 7000 vcr cassetes in that route

File: tetrapod/compass/client.py (hunk 2 above)
Comment (dem4ply) [reply tetrapod/compass/client.py]: thanks for catch that

**Pull request merged**